tests/test_repository.py

```diff
@@ -53,12 +53,16 @@
             "ecryptfs-utils",
             # has a boolean dependency on xwayland
             "at-spi2-core",
-            # has a boolean dependency on hwloc which is not part of SLE
-            "spack",
         ]
         if OS_SP_VERSION >= 6
         else []
     )
+    + (  # has a boolean dependency on hwloc which is not part of SLE
+        ["spack"]
+        if OS_SP_VERSION >= 6
+        and LOCALHOST.system_info.arch in ("aarch64", "x86_64")
```

Comment:
Please surround this in brackets and reformat
```suggestion
        if (OS_SP_VERSION >= 6
        and LOCALHOST.system_info.arch in ("aarch64", "x86_64"))
```

Reply:
Done.